src/instances/concat.yml

```
@@ -0,0 +1,11 @@
+---
+id: concat
+name: >-
+    ConCat lab at TU Berlin
+contact:
+    - name: Peter Kraus
+      email: peter.kraus@tu-berlin.de
+canonical_url: https://datalab.concatlab.eu
+api_url: https://datalab-api.concatlab.eu
```

```suggestion
api_url: https://datalab-api.concatlab.eu
homepage: https://www.tu.berlin/en/concat
```

For some free advertising perhaps?